subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -97,7 +97,7 @@ dependencies {
         api(libs.hikariCP)              { version { strictly("4.0.3"); because("5.x requires Java 11+") }}
         api(libs.httpcore)              { version { strictly("4.4.14") }}
         api(libs.inject)                { version { strictly("1") }}
-        api(libs.ivy)                   { version { strictly("2.3.0"); because("2.4.0 contains a breaking change in DefaultModuleDescriptor.getExtraInfo(), cf. https://issues.apache.org/jira/browse/IVY-1457") }}
+        api(libs.ivy)                   { version { strictly("2.5.2") }}
```

Comment:
❓ I don't understand how that version bump does not require an entry in the verification metadata file. AFAICT we do not trust a key for `org.apache*` or for `org.apache.ivy`

Reply:
I think it's this trusted key is the cause:

```xml
<trusted-key id="ce8075a251547bee249bc151a2115ae15f6b8b72">
   <trusting group="org.apache.ant"/>
   <trusting group="org.apache.commons"/>
   <trusting group="xmlunit"/>
   <trusting group="^org[.]apache($|([.].*))" regex="true"/>
</trusted-key>
```

Reply:
🤦 Of course, the regex 😉 
And it makes sense since the Ant team is maintaining Ivy in the Apache Foundation for now.